pyproject.toml

```diff
@@ -590,14 +590,12 @@ module = [
     "sentry.services.hybrid_cloud.organization_mapping.*",
     "sentry.services.hybrid_cloud.organizationmember_mapping.*",
     "sentry.services.hybrid_cloud.orgauthtoken.*",
-    "sentry.services.hybrid_cloud.pagination",
```

Comment:
Did we want to add the new module locations to the stronglist?

Reply:
ideally it would be `sentry.hybridcloud.*` but it's not passing at the moment

Reply:
I'll get the issues there solved and get it added to the stronglist.